types.go

```diff
@@ -55,6 +55,11 @@ type (
 		// Checksum ensures the integrity of packages
 		// refereced by URL. Ignored for literals.
 		Checksum Checksum `json:"checksum"`
+	}
+
+	PackageRef struct {
+		PackageName      string
+		PackageNamespace string
```

Comment:
In that case shouldn't the actual PackageRef contain solely a UID for identification, to ensure it only reference one version/instance of the VersionedPackage? Potentially you could still add name and namespace for explainability/human-readability.

Ah, so the entrypoint points identifies a single function in a package with (potentially) multiple functions. Seems to me like this belongs more in the FunctionSpec or a separate PackageFunctionRef or something similar, as I would expect something called a `PackageRef` to be just a reference to a package.

Aside, wouldn't it be useful to have a list of all containing functions/files in the PackageSpec? That would allow Fission to easily check if a reference to a package/function is valid.